src/license/license.go

```diff
@@ -75,6 +76,15 @@ func CheckLicenseSecret() error {
 	return nil
 }
 
+func (t *Token) checkTokenExpiration() error {
+	if t.checkExpiration() {
+		err := fmt.Errorf("license is expired, valid license doesn't exist with user-id: %s, key: %s and platform uuid: %s", LCfg.Lcs.UserId, LCfg.Lcs.Key, LCfg.Lcs.PlatformUUID)
```

Comment:
Do we need to print key in the error ?
We need only user and uuid to generate new token. I think only those 2 are required.